Decisions.TruBot/Data/TruBotAssignmentHandler.cs

```diff
@@ -0,0 +1,120 @@
+using System.Net.Http.Json;
+using Decisions.TruBot.Api;
+using DecisionsFramework;
+using DecisionsFramework.Data.ORMapper;
+using DecisionsFramework.Design.ConfigurationStorage.Attributes;
+using DecisionsFramework.ServiceLayer.Actions;
+using DecisionsFramework.ServiceLayer.Actions.Common;
+using DecisionsFramework.ServiceLayer.Services.Assignments;
+using DecisionsFramework.ServiceLayer.Utilities;
+
+namespace Decisions.TruBot.Data
+{
+    [Writable]
+    public class TruBotAssignmentHandler : IAssignmentHandler
```

Comment:
What's the reason we need to have a full Assignment Handler?

Reply:
Is there an alternative to this?